src/ImageSharp/PixelFormats/ColorBuilder{TPixel}.cs

```diff
@@ -76,7 +76,10 @@ public static TPixel FromRGBA(byte red, byte green, byte blue, byte alpha)
         /// </returns>
         private static string ToRgbaHex(string hex)
         {
-            hex = hex.StartsWith("#") ? hex.Substring(1) : hex;
+            if (hex[0] == '#')
```

Comment:
this will fail if `hex` is an empty string. we should be checking the length > 0 too. Probably should be defensive over nulls as well.